From: Logan Gunthorpe <logang@deltatee.com>
Date: Sat, 20 Jul 2013 18:17:14 +0200
Subject: Fix recover CDF option

Upstream (sourceforge) Tracker id 2018613 (See Bug link)
New upstream bug id 39.

Bug: http://sourceforge.net/p/bandwidthd/bugs/39/
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=660810
Forwarded: yes
---
 bandwidthd.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/bandwidthd.c b/bandwidthd.c
index 7d40c0c..bc34aa4 100644
--- a/bandwidthd.c
+++ b/bandwidthd.c
@@ -1192,10 +1192,10 @@ void RCDF_Load(FILE *cdf)
 
         if (fscanf(cdf, "%llu,%llu,%llu,%llu,%llu,%llu,%llu,%llu,",
             &ip->Send.total, &ip->Send.icmp, &ip->Send.udp,
-            &ip->Send.tcp, &ip->Send.ftp, &ip->Send.http, &ip->Send.mail, &ip->Send.p2p) != 7
+            &ip->Send.tcp, &ip->Send.ftp, &ip->Send.http, &ip->Send.mail, &ip->Send.p2p) != 8
           || fscanf(cdf, "%llu,%llu,%llu,%llu,%llu,%llu,%llu,%llu",
             &ip->Receive.total, &ip->Receive.icmp, &ip->Receive.udp,
-            &ip->Receive.tcp, &ip->Receive.ftp, &ip->Receive.http, &ip->Receive.mail, &ip->Receive.p2p) != 7)
+            &ip->Receive.tcp, &ip->Receive.ftp, &ip->Receive.http, &ip->Receive.mail, &ip->Receive.p2p) != 8)
 			goto End_RecoverDataFromCdf;		
 		}
 
